src/main/java/eu/dissco/backend/controller/DigitalMediaObjectController.java

```diff
@@ -101,21 +101,11 @@ public ResponseEntity<List<Integer>> getDigitalMediaVersions(
   @GetMapping(value = "/{prefix}/{postfix}/{version}", produces = MediaType.APPLICATION_JSON_VALUE)
   public ResponseEntity<DigitalMediaObject> getDigitalMediaObject(
       @PathVariable("prefix") String prefix, @PathVariable("postfix") String postfix,
-      @PathVariable("version") int version) {
+      @PathVariable("version") int version) throws JsonProcessingException, NotFoundException {
     var id = prefix + '/' + postfix;
     log.info("Received get request for digital media: {} with version: {}", id, version);
     var digitalMedia = service.getDigitalMediaVersion(id, version);
     return ResponseEntity.ok(digitalMedia);
   }
 
-  @GetMapping(value = "/{prefix}/{postfix}/{version}/json", produces = MediaType.APPLICATION_JSON_VALUE)
```

Comment:
Why delete this?

Reply:
Ah yes needed a bit of soul-searching for this one but I remember. It gets the version for the digitalMediaObject. However, all the logic was in the repository and getting the version was removed for it. I made the functionality to get the version generic in the mongodbRepo. I didn't wanted anymore scope creep so to convert the generic setup to JSON:API I created a new userstory. As all other controllers didn't have JSON:API for this yet I decided to remove it here.